src/coord/src/coord.rs

```diff
@@ -199,6 +208,12 @@ pub struct Coordinator {
     active_conns: HashMap<u32, ConnMeta>,
     /// Map of all persisted tables.
     persisted_tables: Option<PersistentTables>,
+
+    /// Holds pending compaction messages to be sent to the dataflow workers. When
+    /// `since_handles` are advanced or `txn_reads` are dropped, this can advance.
+    since_updates: Rc<RefCell<HashMap<GlobalId, Antichain<Timestamp>>>>,
```

Comment:
I may try and convince you at some point that a `ChangeBatch` rather than an `Antichain` might be interesting here, but I don't think it should hold anything up (nor am I certain it is better, but I should explain at least).

Reply:
I would benefit from an explanation. Are you able to summarize one here? I tried changing this and the fallout was larger than I had hoped, and since I'm unsure of the value of the change, it's not clear whether I should update the other things to compensate.

Reply:
i would also benefit from the explanation